instructions.json

```diff
@@ -0,0 +1,19 @@
+{"instructions": 
+  {
+    "db_name":"apidb",
+	"user_name" : "user",
+	"password" : "password",
+	"region_name": "New Zealand",
```

Comment:
A good first unit test to include could be checking what happens when instructions.json file has an incorrect `db_name` or the postgres db isn't running. We would probably expect some errors to be shown to the user and the program would not allow you to continue. I can give you some help with starting unit testing and I think you should do this on this PR